Cython/Compiler/Nodes.py

```diff
@@ -661,7 +661,7 @@ def analyse(self, return_type, env, nonempty=0, directive_locals=None, visibilit
                 type_node = directive_locals[name]
                 other_type = type_node.analyse_as_type(env)
                 if other_type is None:
-                    error(type_node.pos, "Not a type")
+                    error(type_node.pos, "Not a type in CFuncDeclaratorNode")
```

Comment:
My guess is that this is for debugging (like the `print()` calls that you left in), but just in case, since these are user visible messages, we shouldn't leak internal jargon into them.

Reply:
Fixed.

BTW is there any easier way the check which test cases failed on CI from the log on travis ?